helix-core/src/main/java/org/apache/helix/model/RESTConfig.java

```diff
@@ -83,4 +85,18 @@ public String getBaseUrl(String instance) {
 
     return baseUrl.replace("*", instanceVip);
   }
+
+  /**
+   * Retrieves the configured health URL if no wildcard is present.
```

Comment:
I would suggest include more detail about aggregated healthcheck url and individual health check url. What are the differences, example etc.